emc2/core/model.py

```diff
@@ -213,6 +232,17 @@ def __init__(self, file_path):
             # No need for lat and lon dimensions
             self.ds = self.ds.squeeze(dim=('lat', 'lon'))
 
+        # crop specific model output time range (if requested)
+        if time_range is not None: 
+            try:
+                if np.issubdtype(time_range.dtype, np.datetime64):
+                    super()._crop_time_range(time_range)
+                else:
+                    raise RuntimeError("input time range is not in the required datetime64 data type")
+            except:
```

Comment:
You'll have to find out here what type of error you want to catch. I highly recommend against using a blanket except statement here and PEP8 does not allow for a blanket except statement.